### PR TITLE
chore: renames pmsEmails to contactEmails

### PR DESCRIPTION
**Pull request by markcsims**
renames pmsEmails to contactEmails

### CODE REVIEW

File: apps/frontend/src/shared-research/__tests__/export.test.ts [hunk 1]
```diff
@@ -137,13 +137,13 @@ describe('researchOutputToCSV', () => {
   it('flattens and orders pm emails', () => {
```
Comment (kmaid) [inline apps/frontend/src/shared-research/__tests__/export.test.ts:137]: This test name should also change

File: packages/react-components/src/templates/__tests__/SharedResearchOutput.test.tsx [hunk 2]
```diff
@@ -80,10 +80,12 @@ it('displays access instructions when data provided', () => {
 
 it('displays contact pm card when there are contact emails', () => {
```
Comment (kmaid) [inline packages/react-components/src/templates/__tests__/SharedResearchOutput.test.tsx:81]: Test name

File: packages/react-components/src/templates/__tests__/SharedResearchProposal.test.tsx [hunk 3]
```diff
@@ -68,11 +68,11 @@ it('falls back to created date when added date omitted', () => {
 
 it('displays contact pm card when there are contact emails', () => {
```
Comment (kmaid) [inline packages/react-components/src/templates/__tests__/SharedResearchProposal.test.tsx:69]: Test name

**Review by kmaid — APPROVED** [review]

File: packages/react-components/src/templates/__tests__/SharedResearchOutput.test.tsx (hunk 2 above)
Comment (markcsims) [reply packages/react-components/src/templates/__tests__/SharedResearchOutput.test.tsx:81]: Leaving this as is, as we are asserting on the word "contact pm"

File: apps/frontend/src/shared-research/__tests__/export.test.ts (hunk 1 above)
Comment (markcsims) [reply apps/frontend/src/shared-research/__tests__/export.test.ts:137]: fixed

File: packages/react-components/src/templates/__tests__/SharedResearchProposal.test.tsx (hunk 3 above)
Comment (markcsims) [reply packages/react-components/src/templates/__tests__/SharedResearchProposal.test.tsx:69]: Leaving this as is, as we are asserting on the word "contact pm"

**Review by kmaid — APPROVED** [review]

**Pull request merged**